starter-code/models/User.js

```diff
@@ -0,0 +1,25 @@
+const mongoose = require("mongoose");
+
+const userSchema = new mongoose.Schema(
```

Comment:
Muy bien chicos, empiezo a ver que hacéis `Schemas` más completos utilizando la validación de `Mongoose`. De todas formas se pueden realizar aún más validaciones sobre todo en el campo `password` (longitud mínima, que tenga mayúscula, un número, una letra, ...)